README.md

```diff
@@ -266,11 +266,7 @@ The following flags are available for `organizations get`:
 * Get an organization and include its labels `ocm support get organizations [organizationID] --fetchLabels --fetchCapabilities`
 * Get an organization and include its capabilities `ocm support get organizations [organizationID] --fetchCapabilities`
 * Get an organization and include its quota: `ocm support get organizations [organizationID] --fetchQuota`
-<<<<<<< HEAD
-* Get an organization and include its resource quota: `ocm support get organizations [organizationID] --fetchQuota`
-=======
 * Get an organization and include its resource quota: `ocm support get organizations [organizationID] --fetchSkus`
```

Comment:
Can we have as: "Get an organization and include its SKUs"? The reason for that is that the term SKU is used more often.